framework/tester.py

```diff
@@ -328,3 +328,15 @@ def wait_while_busy(self, *items):
         for item in items:
             if item.is_running():
                 item.wait_for_finish()
+
+    # Should replace all duplicated instances of wait_all_connections
+    def wait_all_connections(self, tmt=1):
+        for sid in self.__servers:
+            srv = self.__servers[sid]
+            if not srv.wait_for_connections(timeout=tmt):
+                return False
+        return True
```

Comment:
Good change! There aren't so many instances of the method, so could you please replace them with the method? Or, worse, just add TODO comment to all of them. Otherwise we'll just live with one more duplication of the function :)